src/applications/TrainingModule.cpp

```diff
@@ -420,18 +422,18 @@ VectorDouble TrainingModule::CrossValidation(const VariableSizeMatrixType inputF
 
     std::vector<int> FinalSelectedFeatures;
     VectorDouble crossvalidatedAccuracies;
-    if (featureselectiontype == 1)
-      FinalSelectedFeatures = EffectSizeBasedFeatureSelection(scaledFeatureSet, traininglabels, classifiertype, crossvalidatedAccuracies);
+    //if (featureselectiontype == 1)
+    //  FinalSelectedFeatures = EffectSizeBasedFeatureSelection(scaledFeatureSet, traininglabels, classifiertype, optimizationtype,cvtype, crossvalidatedAccuracies);
     //else if (featureselectiontype == 2)
     //  FinalSelectedFeatures = CorrelationBasedFeatureSelection(scaledFeatureSet,traininglabels,classifiertype);
-    else if (featureselectiontype == 3)
-      FinalSelectedFeatures = SVMFFSBasedFeatureSelection(scaledFeatureSet, traininglabels, classifiertype);
+    //else if (featureselectiontype == 3)
+    //  FinalSelectedFeatures = SVMFFSBasedFeatureSelection(scaledFeatureSet, traininglabels, classifiertype,crossvalidatedAccuracies);
     //else if (featureselectiontype == 4)
     //  FinalSelectedFeatures = SVMRFEBasedFeatureSelection(scaledFeatureSet,traininglabels,classifiertype);
-    else
-    {
-      std::cout << "The selected feature selection method is not avaialble inside CaPTk. Please select another method..." << std::endl;
-    }
+    //else
+    //{
+    //  std::cout << "The selected feature selection method is not avaialble inside CaPTk. Please select another method..." << std::endl;
+    //}
```

Comment:
Please delete commented code if not needed, otherwise please add specific comment as to why it was commented.

Reply:
i will do many changes in this section and will add comments before finalizing. It is fine to keep it like that for now. keeping this thread open for your opinion.

Reply:
Sure. This is work in progress.